Makefile

```diff
@@ -20,6 +20,8 @@ SCYLLA_VERSION?=scylla:6.0.1
 IP_FAMILY?=IPV4
 RAFT_SCHEMA?=none
 TABLETS?=enabled
+# if true starts the scylla cluster with ssl only config
+SSL_ENABLED?=false
```

Comment:
I would say that we should have it enabled by default, as this is the more common user setup.
This brings me to our CI - the current checks passed only for disabled SSL.
It would be good if they run on both enabled and disabled SSL.
Please take a look at our gh actions config:
- https://github.com/scylladb/scylla-manager/blob/master/.github/actions/test-setup/action.yml sets up testing env
- https://github.com/scylladb/scylla-manager/tree/master/.github/cfg generates workflows based on the config file

Workflows should support `SSL_ENABLED` input. For most workflows it should be set to true (as in default), but we can set it to false in some of them. Note that we can't test the whole cofig matrix, as it would require too many workflows, but it's enough to have some cases with `SSL_ENABLED` set to false. For example:

```yam
...
- scylla-version: scylla-enterprise-nightly:latest-enterprise
  ip-family: IPV4
  raft-schema: none
  tablets: disabled
  ssl-enabled: false

- scylla-version: scylla-enterprise:2024.1.12
  ip-family: IPV6
  raft-schema: none
  tablets: none
  ssl-enabled: false
...
```

Reply:
makes sense! I've updated CI and made several fixes to our integration tests, because some of them assumed that the cluster could only be configured without SSL.